### ISSUE TITLE
No notifications on Android device

### ISSUE THREAD

**Issue by pmarini** [nextcloud/spreed#1972]
# Steps to reproduce

I expect a notification when someone sends me a message on Nextcloud Talk application installed on my smartphone.  No notifications show up.

The same behaviour is observed from at least one other Android device.

### Expected behaviour
A notification pops up after receiving a message

### Actual behaviour
No notification pops up after receiving a message

## Browser

**Microphone available:** yes

**Camera available:** yes

**Operating system:** Android 8.0.0

**Browser name:** Not applicable

**Browser version:** Not applicable

### Browser log
Not applicable

## Spreed app

**Spreed app version:** 5.0.4

**Custom TURN server configured:** no

**Custom STUN server configured:** no

## Server configuration

**Operating system**: CentOS 7

**Web server:** Apache

**Database:** 5.5.60-MariaDB MariaDB Server

**PHP version:**

PHP 7.0.33 (cli) (built: Dec  6 2018 22:30:44) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
**Nextcloud Version:** 15.0.10

**List of activated apps:**

$ sudo -u apache php occ app:list
The current PHP memory limit is below the recommended value of 512MB.
Enabled:
  - cloud_federation_api: 0.1.0
  - dav: 1.8.2
  - federatedfilesharing: 1.5.0
  - files: 1.10.0
  - files_versions: 1.8.0
  - lookup_server_connector: 1.3.0
  - nextcloud_announcements: 1.4.0
  - oauth2: 1.3.0
  - password_policy: 1.5.0
  - provisioning_api: 1.5.0
  - serverinfo: 1.5.0
  - spreed: 5.0.4
  - twofactor_backupcodes: 1.4.1
  - updatenotification: 1.5.0
  - workflowengine: 1.5.0
Disabled:
  - accessibility
  - activity
  - admin_audit
  - comments
  - encryption
  - federation
  - files_external
  - files_pdfviewer
  - files_sharing
  - files_texteditor
  - files_trashbin
  - files_videoplayer
  - firstrunwizard
  - gallery
  - logreader
  - notifications
  - sharebymail
  - support
  - survey_client
  - systemtags
  - theming
  - user_ldap

**Nextcloud configuration:**

The current PHP memory limit is below the recommended value of 512MB.
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "80.211.83.105",
            "chat-server",
            "chat.x51-hserver.com"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "15.0.10.0",
        "overwrite.cli.url": "http:\/\/80.211.83.105",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "maintenance": false,
        "theme": "",
        "loglevel": 0,
        "mail_smtpmode": "smtp",
        "updater.release.channel": "stable",
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}

### Server log (data/nextcloud.log)

100MB, too big to put it here

![Screenshot_20190710-104935](https://user-images.githubusercontent.com/7346796/60956922-045e5b00-a304-11e9-965f-7bca5385fca3.jpg)
![Screenshot_20190710-103831](https://user-images.githubusercontent.com/7346796/60956924-045e5b00-a304-11e9-90b9-680789ae8023.jpg)
![Screenshot_20190710-104048](https://user-images.githubusercontent.com/7346796/60956925-045e5b00-a304-11e9-899c-a342491041fc.jpg)

**Comment by nickvergessen** [comment]
Currently notifications are not sent by default for group conversations. You may try to send a message mentioning the user e.g. `@test`

The other thign to note is, that you do not receive a notification when you are in the conversation with any other device or with the browser.

**Comment by pmarini** [comment]
Thanks for the quick reply!

I'm trying right now, with a 1-to-1 conversation

- user pmarini is writing from the Android device mentioned above
- user test_usr is writing from a laptop in Firefox

I would expect a notification on pmarini device at last test_usr message
![image](https://user-images.githubusercontent.com/7346796/60961509-a84c0480-a30c-11e9-807b-8e790259279c.png)

Is that correct? If so, there is somewhere a problem as the notification doesn't pop up.

While I do not exclude that I did some weird configuration on my Android device that prevents notifications from this particular app (other applications show notif. as expected), I'm trying to understand if other users experience the same issue or not.

**Comment by nickvergessen** [comment]
It looks like you don't have the notifications app installed on your nextcloud server? :thinking:

**Comment by nickvergessen** [comment]
After you enabled/installed that app, you need to remove and then newly setup the account on the mobile apps in order to receive push notifications

**Comment by pmarini** [comment]
Thanks a lot @nickvergessen !

In an attempt of minimizing the number of applications running on the Nextcloud server, I have disabled the Notifications app too as I deemed it not useful. I was clearly wrong..

As I thought this server, it was just to chat on Nextcloud Talk: no file sharing, no collaborative editing, etc.

It seems to work now

Let me do some additional tests before closing this ticket, I expect to close it tomorrow.
